syft/execution/plan.py

```diff
@@ -70,7 +70,7 @@ def method2plan(*args, **kwargs):
     )
 
 
-class Plan(AbstractObject, ObjectStorage):
+class Plan(AbstractObject):
```

Comment:
Holy cow, didn't realize this PR would remove `ObjectStorage` as a parent of `Plan`. 👍

Reply:
This could have been removed before but it was still here somehow. Anyway, it's better now :)